webapp/src/components/github_repo_selector/index.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+
+import {getRepos} from '../../actions';
+
+import GithubRepoSelector from './github_repo_selector.jsx';
+
+function mapStateToProps(state) {
+    return {
+        yourRepos: state['plugins-github'].yourRepos,
```

Comment:
All of the previously created `mapStateToProps` and `mapDispatchToProps` are using the hardcoded `plugins-github` instead of the `id` value from the manifest. 

I will keep as is for now and create a HW ticket to use the `id` for the manifest.

Reply:
ticket created https://github.com/mattermost/mattermost-plugin-github/issues/238

and ticket picked up 2 minutes after posting :smile: